nodes/viz/texture_viewer.py

```diff
@@ -220,12 +241,26 @@ def xy_offset(self):
         return int(a[0] + b), int(a[1])
 
     def get_buffer(self):
+        # width=0
+        # height=0
+        size_tex = 0
         # data = self.inputs['Float'].sv_get(deepcopy=False)[0]
         data = np.array(self.inputs['Float'].sv_get(deepcopy=False)).flatten()
+        #### This code below stall the machine ####
```

Comment:
this code is wrong do NOT use it:  the `total_size` of the Buffer with the custom texture size became: 

```
206  *  124 = 
25544

 25544 * 25544 =

652495936

652495936 * 3=
1957487808  Pixels!!!
```
how to kill a computer.... :disappointed: 

I will fix it.